playbooks/cloud-provider-openstack-acceptance-test-lb-octavia/run.yaml

```diff
@@ -89,23 +90,23 @@
             service_name="${test_case}-http-nginx-service"
             # Delete fake floating-network-id to use the default one in cloud config
             sed -i '/loadbalancer.openstack.org/d' "$test_file"
-            "$kubectl" create -f "$test_file"
+            '{{ kubectl }}' create -f "$test_file"
 
             if ! service_name="$service_name" timeout 600 bash -c '
                 while :
                 do
-                    [[ -n $("$kubectl" describe service "$service_name" | awk "/LoadBalancer Ingress/ {print \$3}") ]] && break
+                    [[ -n $({{ kubectl }} describe service "$service_name" | awk "/LoadBalancer Ingress/ {print \$3}") ]] && break
```

Comment:
{{ kubectl }} should be '{{ kubectl }}'

Reply:
Stripped the single quotes is intentional, because the whole while loop has been quoted

Reply:
my fault, thank you.